apps/sms/views/conversation/js/conversation.js

```diff
@@ -1159,7 +1165,8 @@ var ConversationView = {
       // If we're leaving a thread's message view,
       // ensure that the thread object's unreadCount
       // value is current (set = 0)
-      if (Threads.active) {
+      if (this.currentThread) {
+        // TODO: Will need to replace with Conversation service method.
```

Comment:
not sure if this condition will be useful in the future actually -- I think it's here especially because we want the "unread" mark to disappear from the thread list even if we haven't marked the thread as read yet (because it's not finished loading)